packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -47,22 +47,33 @@ const HomePage = () => {
   const { push } = useHistory();
   const { search } = useLocation();
   const isMounted = useIsMounted();
-
-  const { data, dataCount, dataToDelete, isLoading } = reducerState.toJS();
+  const { data, dataCount, dataToDelete, isLoading, searchParams } = reducerState.toJS();
   const pluginName = formatMessage({ id: getTrad('plugin.name') });
   const paramsKeys = ['_limit', '_start', '_q', '_sort'];
   const debouncedSearch = useDebounce(searchValue, 300);
 
   useEffect(() => {
     handleChangeParams({ target: { name: '_q', value: searchValue } });
+
```

Comment:
`debouncedSearch` should be use instead of `searchValue` so you can remove the eslint-lint override.